servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/WriteStreamSubscriber.java

```diff
@@ -67,6 +72,7 @@
  */
 final class WriteStreamSubscriber implements PublisherSource.Subscriber<Object>, ChannelOutboundListener, Cancellable {
     private static final Logger LOGGER = LoggerFactory.getLogger(WriteStreamSubscriber.class);
+    private static final Object WRITE_BOUNDARY = new Object();
```

Comment:
Consider making this a `GenericFutureListener`. This was can tighten up the generics for the queue.

Reply:
good call